src/scripts.js

```diff
@@ -1,22 +1,24 @@
-// GLOBAL
+// IMPORTS
 import './styles.css';
 import apiCalls from './apiCalls';
 import ingredientsData from '../src/data/ingredients';
 import recipeData from '../src/data/recipes';
-
 import RecipeRepository from '../src/classes/RecipeRepository';
 import Recipe from '../src/classes/Recipe';
 
-const recipeRepository = new RecipeRepository(recipeData);
+// GLOBAL VARIABLES
+const recipeList = recipeData.map(recipe => {
+  return new Recipe(recipe);
+});
+
+const recipeRepository = new RecipeRepository(recipeList)
+let recipeCards = []
```

Comment:
I think at some point we could look into this more, but since it's working I think we can wait to circle back to it during our final refactor. It feels like there should be some sort of way to not have this as a global variable since its only needed in the display recipes section.